### PR TITLE
 Update the markdown file to reflect the current implementation of histogram.

### PR DESCRIPTION
**Pull request by Yun-Ting**
## Changes
Update the upper bounds of intervals to be inclusive.

### CODE REVIEW

File: docs/metrics/getting-started-histogram/README.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ output from the console, similar to shown below:
...
-(0 - 5) : 0
-(5 - 10) : 0
-(10 - 25) : 2
-(25 - 50) : 0
-(50 - 75) : 1
-(75 - 100) : 1
-(100 - 250) : 6
-(250 - 500) : 18
-(500 - 1000) : 34
+(-? - 0] : 0
```
Comment (reyang) [inline docs/metrics/getting-started-histogram/README.md:37]: I think it is something like this (in the latest main branch):

```text
(-Infinity, 0]: 0
(0, 5]: 2
(5, 10]: 1
(10, 25]: 5
(25, 50]: 7
(50, 75]: 5
(75, 100]: 8
(100, 250]: 40
(250, 500]: 90
(500, 1000]: 155
(1000, +Infinity): 0
```

Comment (Yun-Ting) [reply docs/metrics/getting-started-histogram/README.md:37]: Yes, you are right. I was unsure whether the -Infinity/+Infinity representation was due to my local encoding or not. Let me push a new iteration. Thanks!

Comment (reyang) [reply docs/metrics/getting-started-histogram/README.md:37]: I think [this PR](#2285) changed it to use ASCII output of IEEE754 `+Infinity` and `-Infinity`, so we've avoided the code page issue. https://github.com/open-telemetry/opentelemetry-dotnet/blob/e27070d2204b82a1f19c43524ab4ed4394d3a3f3/src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs#L96

File: docs/metrics/getting-started-histogram/README.md [hunk 2]
```diff
@@ -34,17 +34,17 @@ output from the console, similar to shown below:
 ```text
 Export 14:30:58.201 14:30:59.177 histogram [tag1=value1;tag2=value2] Histogram, Meter: TestMeter/0.0.1
```
Comment (reyang) [inline docs/metrics/getting-started-histogram/README.md:35]: The timestamp is updated to use the [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, so the output would look like:

```text
Export (2021-09-02T00:47:54.9655496Z, 2021-09-02T00:48:00.0963511Z] histogram tag1=value1;tag2=value2 Histogram, Meter: TestMeter/0.0.1
Value: Sum: 159160 Count: 313
(-Infinity, 0]: 0
(0, 5]: 2
(5, 10]: 1
(10, 25]: 5
(25, 50]: 7
(50, 75]: 5
(75, 100]: 8
(100, 250]: 40
(250, 500]: 90
(500, 1000]: 155
(1000, +Infinity): 0
```

**Review by reyang — APPROVED** [review]
LGTM assuming the comments are taken.

File: docs/metrics/getting-started-histogram/README.md (hunk 2 above)
Comment (Yun-Ting) [reply docs/metrics/getting-started-histogram/README.md]: Thanks for pointing me to the reference! 
I did a copy paste from console this time, should be accurate now. :-)

File: docs/metrics/getting-started-histogram/README.md [hunk 3]
```diff
@@ -32,19 +32,19 @@ output from the console, similar to shown below:
...
-(25 - 50) : 0
-(50 - 75) : 1
-(75 - 100) : 1
-(100 - 250) : 6
-(250 - 500) : 18
-(500 - 1000) : 34
-(1000 - ?) : 0
+Export (2021-09-02T01:02:09.8013446Z, 2021-09-02T01:02:19.9749573Z] histogram tag1=value1;tag2=value2 Histogram, Meter: TestMeter/0.0.1
+Value: Sum: 304929 Count: 623
+(-Infinity, 0]: 0
```
Comment (ThomsonTan) [inline docs/metrics/getting-started-histogram/README.md:37]: A random ask, why the ranges are not inclusive for +/-Infinity? I think both has valid representation in IEEE854?

Comment (cijothomas) [reply docs/metrics/getting-started-histogram/README.md:37]: https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L437-L439

Comment (utpilla) [reply docs/metrics/getting-started-histogram/README.md:37]: Infinity is undefined, right? (at least mathematically). We cannot have an actual number which is equal to infinity or -ve infinity.

**Review by utpilla — APPROVED** [review]

File: docs/metrics/getting-started-histogram/README.md [hunk 4]
```diff
@@ -32,19 +32,19 @@ output from the console, similar to shown below:
...
+(0, 5]: 5
+(5, 10]: 5
```
Comment (cijothomas) [inline docs/metrics/getting-started-histogram/README.md:39]: While this representation is what we want, the current main branch is doing upperbound as exclusive:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Metrics/MetricAggregators/HistogramMetricAggregator.cs#L147-L151

This is addressed in the upcoming refactor of Aggregator. Lets treat that as merely a bug, and okay to merge this PR :)

**Review by cijothomas — APPROVED** [review]

**Pull request merged**